### ISSUE TITLE
Doc-comments inside using groups 

### ISSUE THREAD

**Issue by DarkEld3r** [rust-lang/rust#51985]
The following is currently not permitted:
```rust
pub use third_party::{
    /// Doc-comment explaining reexported entity.
    some_function,
    // Normal comments are allowed, though.
    some_other_function,
};
```

I'm very fond of group usings, so I prefer to use them everywhere, but it is impossible in situations where the documentation is needed (we have encountered this problem [here](https://github.com/exonum/exonum/pull/756/files#diff-db51f3139619c4526a5967fac3ee28d9R42), by the way).

**Comment by Havvy** [comment]
I think this requires a dedicated RFC to outer attributes on that location.

**Comment by DarkEld3r** [comment]
@Havvy Makes sense, but I'm not ready to take it on myself. I hope someone will be interested in writing an RFC.

**Comment by QuietMisdreavus** [comment]
We don't even display docs on `use` statements to begin with, so that would need to happen as well.

I don't think allowing attributes inside a grouped `use` statement requires an RFC, per se, but it will need to be explicitly added. @rust-lang/compiler?

**Comment by cramertj** [comment]
+1 for allowing attributes on use-statement subitems/groups.

**Comment by estebank** [comment]
I'm intrigued at what docs on imports would look like in the rendered docs. I can see the point for reexports (`pub use`), but not for the general case (other than in the code itself, at which point they are just comments).

**Comment by fmease** [comment]
Closing in favor of this new issue https://github.com/rust-lang/rust/issues/141704 from 2025 because that seems to have a lot more momentum & T-lang involvement.
